about.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Gender Equality</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="stylesheet" href="./CSS/about.css">
+</head>
+<body>
+  <header class="header">
+    <div class="social_media">
+      <ul class="media-nav d-flex flex-center">
+        <li class="media_link"><a href="https://web.facebook.com/asia.magana"><i class="fa fa-facebook"></i></a></li>
+        <li class="media_link"><a href="https://twitter.com/asia_magana"><i class="fa fa-twitter"></i></a></li>
+        <li class="media_link"><a href="#">English</a></li>
+        <li class="media_link"><a href="#">My page</a></li>
+        <li class="media_link"><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <nav class="menu">
+      <div class="hamburger d-flex flex-center">
+        <img src="./images/icons/Union.png" class="show-menu-btn" alt="union">
+        <img src="./images/icons/Icon@2x.svg" class="closed-menu-btn" alt="">
+        <div class="logo">
+          <img src="./images/gender_equality_2.png" alt="logo">
+        </div>
+        <ul class="nav_menu d-flex">
+          <li class="menu_link"><a href="./index.html">Home</a></li>
+          <li class="menu_link active"><a href="#About">About us</a></li>
+          <li class="menu_link"><a href="#Program">Mission</a></li>
+          <li class="menu_link"><a href="#Team">Team</a></li>
+          <li class="menu_link"><a href="#Partner">Partners</a></li>
+          <li class="menu_link active"><a href="./index.html">CloudFlare</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <main>
+    <section id="About" class="home_section">
+      <div class="home_section_title">
+          <h2>"Hello! Sharing world"</h2>
+          <h1> GLOBAL ISSUES: GENDER EQUALITY AND<br>WOMEN'S EMPOWERMENT</h1>
```

Comment:
- [ ] Please be careful with the size of your containers and text, because right now this text is having some scrolling because it is bigger than the container itself, please work on this issue.